src/pages/_app.tsx

```diff
@@ -15,7 +15,19 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
     Router.events.on("routeChangeStart", () => setIsLoading(true));
     Router.events.on("routeChangeComplete", () => setIsLoading(false));
     Router.events.on("routeChangeError", () => setIsLoading(false));
-  }, [Router.events]);
+
+    const unobtrusiveDataReload = () => {
+      Router.replace(window.location, undefined, { scroll: false }).catch(
+        () => {
+          throw new Error("Could not do unobtrusiveDataReload");
+        }
+      );
+    };
+    const i = setInterval(unobtrusiveDataReload, 7000);
```

Comment:
Fetching every 7 seconds seems very aggressive.

Reply:
Yeah it's just that often for demo and debugging.